webcompat/static/js/lib/issues.js

```diff
@@ -145,41 +132,6 @@ issues.AsideView = Backbone.View.extend({
   }
 });
 
-issues.BodyView = Backbone.View.extend({
-  mainView: null,
-  initialize: function(options) {
-    this.mainView = options.mainView;
-  },
-  render: function() {
-    // hide metadata
-    var issueDesc = $(".js-Issue-markdown");
-
-    issueDesc
-      .contents()
-      .filter(function() {
-        //find the bare html comment-ish text nodes
-        return this.nodeType === 3 && this.nodeValue.match(/<!--/);
-        //and hide them
-      })
-      .wrap('<p class="is-hidden"></p>');
-
-    // this is probably really slow, but it's the safest way to not hide user data
-    issueDesc
-      .find("p:last-of-type em:contains(From webcompat.com)")
-      .parent()
-      .addClass("is-hidden");
-
-    if (this.mainView._isNSFW) {
```

Comment:
This is something, we would probably have to cater for in the server side rendering. To note that the request of the JSON gives us already the tag with the NSFW.

Reply:
> This is something, we would probably have to cater for in the server side rendering

Would you like me to do that in this PR?
Or should I just add it back now?

Reply:
whichever is 
1. simpler for you
2. effective for the code

Probably aim for now for the JS if it's simple and straightforward and we can still open a second issue to improve it if it's faster on the server side.

Reply:
Okay. I'll explore a bit around this and decide if it's worth it to do it now.